js/eclipsefdn.adopters.js

```diff
@@ -81,7 +81,8 @@
     });
   }
 
-  function fireCall(opts, callback, currentData = []) {
+  function fireCall(opts, callback, currentData) {
+    var currentData = currentData || [];
```

Comment:
we shouldn't set overtop of parameter names, we should just have a different var name here